src/analyses/tutorials/unitAnalysis.ml

```diff
@@ -3,7 +3,9 @@
 open Prelude.Ana
 open Analyses
 
-module Spec : Analyses.MCPSpec with module D = Lattice.Unit and module C = Lattice.Unit =
+(* module Spec : Analyses.MCPSpec with module D = Lattice.Unit and module C = Lattice.Unit and type marshal = unit = *)
+(* No signature so others can override module G *)
+module Spec =
```

Comment:
Honestly, I think here duplication is less bad then not having a signature here, because that tends to be very helpful for newcomers such that they get specific error messages for some functions instead of a general "Module does not match" message that is complicated to read.

At least that was my impression when teaching 10 people how to use Goblint at the ConVEY retreat in Venice.

Reply:
But before this PR, on master there was the signature at the time. Also, having the signature here doesn't really make the error message simpler: if you've mistyped a transfer function, then the entire module still gets underlined, not just that function, and the command-line compiler error is literally the same.

Reply:
Ok, if the error that is reported is the same without the signature, I guess it doesn't make a difference. I seem to recall people getting more local error messages with the signature as opposed to without it, but maybe I am mistaken.

Reply:
I think the localized errors you get would be due to having type annotations on all these transfer function arguments, as long as they are right. Then if you mistype something within the function, it's localized to just that function, not the entire module, because it's typechecked more locally.

In that regard, these transfer functions are good, but in various analyses we don't have all the arguments annotated like that.